app/geo/gui/src/ts/twxplore/gui/geo/states/map/MapFeatureAttributeState/MapNumericFeatureAttributeState.ts

```diff
@@ -8,7 +8,7 @@ This is important info that will be used for setting filter values.
 Unsure, if I should call this FeatureTypeAttributeState instead.
 Or maybe 'AttributeOfFeatureTypeState'
 */
-export interface MapFeatureAttributeState {
+export interface MapNumericFeatureAttributeState {
```

Comment:
I like this specialization. Do all of these fields have to be null? Can you make the object undefined until you can populate everything?

In general, if two fields must be populated together or neither populated, then you should make them a separate interface + single member on the parent interface. I suspect that's the case with min and max here. Either you have them both or you have neither.